.eslintrc

```diff
@@ -28,5 +28,10 @@
     "no-console": "off",
     "react/jsx-uses-vars": 2,
     "react/prop-types": 0
+  },
+  "settings": {
```

Comment:
I was getting a warning in `npm run lint` that eslint was unable to detect react version. I refered to [this](https://github.com/yannickcr/eslint-plugin-react/issues/2157#issuecomment-473680689)

Reply:
Is it coming in the new version of react?
I didnt face this may be because I did long time ago. Anyway this looks good 👍

Reply:
It appeared as a warning in the Travis pipeline, so I wanted to ensure that there are least or almost no warning/s in the pipeline